sphinx/domains/cpp.py

```diff
@@ -3126,7 +3126,7 @@ def get_id(self, version: int, objectType: str = None,
                 res.append(symbol.get_full_nested_name().get_id(version))
             else:
                 print(objectType)
-                assert False
+                raise AssertionError()
```

Comment:
As above, please change to something similar to ``raise AssertionError(objectType)``.